routes/golfer_profile.go

```diff
@@ -29,6 +29,9 @@ func golferGET(w http.ResponseWriter, r *http.Request) {
 
 	db := session.Database(r)
 	golfer := session.GolferInfo(r).Golfer
+	golferInfo := session.GolferInfo(r)
```

Comment:
So calling `session.GolferInfo` a second time will do another redundant SQL select. Can we not do something like
```go
golfer := session.GolferInfo(r)
```
I suspect all the callers wouldn't care that they're getting the superset, I honestly don't recall why I specifically reduce it to the subset struct.